stubs/dateparser/dateparser/timezone_parser.pyi

```diff
@@ -16,3 +16,4 @@ def build_tz_offsets(search_regex_parts): ...
 def get_local_tz_offset(): ...
 
 local_tz_offset: Any
+timezone_info_list: list[dict[str, list[str | tuple[str, ...]]]]
```

Comment:
```suggestion
```

A re-export of https://github.com/python/typeshed/blob/c6126b5f52c9e1ad294f7435d1a2242c2f14103a/stubs/dateparser/dateparser/timezones.pyi#L3 (Though it would be great if you could put the new, improved type there!)